src/main/java/com/softserveinc/dokazovi/controller/UserController.java

```diff
@@ -62,6 +65,18 @@
 				.body(userService.findAllExpertsByDirectionsAndRegions(directions, regions, pageable));
 	}
 
+	@GetMapping(USER_ALL_EXPERTS_BY_NAME + USER_NAME)
+	@ApiPageable
+	@ApiOperation(value = "Get experts ordered by name coincidences. Default 5 per page.")
+	public ResponseEntity<Page<UserDTO>> getAllExpertsByNameOrderedByRequest(
+			@PageableDefault(size = 5) Pageable pageable,
```

Comment:
`@PageableDefault(size = 5) Pageable pageable`
This is a single search request. Pageable is not needed in this response